tests/run_examples.py

```diff
@@ -197,7 +205,9 @@ def get_exe_cmd(ex, nodist, nostream):
             req_version["sycl/" + os.path.basename(ex)], get_daal_version()
         ):
             return None
-        if not check_device(req_device["sycl/" + os.path.basename(ex)], availabe_devices):
+        if not check_device(
```

Comment:
Maybe one final question. What does `check_device` do? It's such a bad name, I don't understand what it's supposed to happen. Can we rename it or at least add a docstring to the function that helps me understand what's going on?